src/APIFunctions/3DPrinting.js

```diff
@@ -99,7 +111,7 @@ export async function delete3DPrintRequest (requestToDelete, token) {
  * @returns {boolean} - If the request was sucessfully deleted or not.
```

Comment:
Be sure to update the documentation of this comment, we don't return a boolean anymore

```suggestion
 * @returns {ApiResponse} Containing any error information related to the 
 * update
```